uv.lock

```diff
@@ -137,11 +137,11 @@ wheels = [
 
 [[package]]
 name = "cachetools"
-version = "5.5.0"
+version = "5.5.1"
```

Comment:
how come this all got bumped?

Reply:
My experimental branch included a sync because I ran into some buggy behaviour (I don't remember what). I'll drop this change.